src/components/doi/doi.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled';
+import { match } from 'ts-pattern';
+
+import doiSvg from './svg/doi.svg';
```

Comment:
I think you need to inline the svg (can be in its own component file), or it will not work in SSR because Node.js cannot import `.svg`